tools/ackdiscover/printer.py

```diff
@@ -11,15 +11,13 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-import dataclasses
+from dataclasses import dataclass, field, asdict
```

Comment:
I dislike importing objects. Much prefer to stick with only importing modules... This makes it much more clear in the Python code following where a particular object is defined...